src/reviewList.jsx

```diff
@@ -1,17 +1,23 @@
-import { useState } from "react"
+import { useEffect, useState } from "react"
 function ReviewList(review_id){
 const [reviews, setReviews] = useState([])
-    fetch('https://jamies-backend-project.onrender.com/api/reviews')
+const [isLoading, setIsLoading] = useState(true)
+   
+useEffect(()=>{
+    setIsLoading(true)
+fetch('https://jamies-backend-project.onrender.com/api/reviews')
```

Comment:
We want to be separating fetch requests out to an api file where we have a base url and add on to it depending on the data we are fetching :)